setup.py

```diff
@@ -1,16 +1,21 @@
+"""
+A full-screen image modal XBlock
+"""
 import json
 from os import path
 from setuptools import setup
 
 
+version = '0.5.0'
```

Comment:
This is the main thing that will be updated frequently;
let's make it easier by putting it up top.